crypto/conf/conf_def.c

```diff
@@ -732,6 +733,30 @@ static BIO *get_next_file(const char *path, OPENSSL_DIR_CTX **dirctx)
 }
 #endif
 
+static int is_keytype(const CONF *conf, char c, unsigned short type)
+{
+    const unsigned short * keytypes = (const unsigned short *) conf->meth_data;
+    unsigned char key = (unsigned char)c;
+
+#ifdef CHARSET_EBCDIC
+# if CHAR_BIT > 8
+    if (key > 255) {
+        /* key is out of range for os_toascii table */
+        return 0;
+    }
+# endif
```

Comment:
What makes you think that >8bitchars is a ebcdic only thing?

Reply:
Of course it's not. But the `key > 255` check is only needed to prevent an out of bound access to the ebcdic-to-ascii table. Below there is another check for `key > 127`.